teams/docs.toml

```diff
@@ -23,3 +23,7 @@ extra-teams = ["reference"]
 [[lists]]
 address = "doc-team@rust-lang.org"
 extra-teams = ["reference"]
+
+[[lists]]
+address = "doc-team@rust-lang.org"
+extra-teams = ["rust-by-example"]
```

Comment:
Pretty sure you can just add this to the existing `doc-team@rust-lang.org` section:

```
extra-teams = ["reference", "rust-by-example"]
```

Reply:
Fixed! :)